### PR TITLE
Added Multiple Websites

### PR DESCRIPTION
**Pull request by aCrazyPOTATO**
- Nepal Police
- Supreme Court

### CODE REVIEW

**Review by diwash007 — DISMISSED** [review]
GG

File: data/websites.json [hunk 1]
```diff
@@ -49,6 +49,17 @@
       "name": "Office of the Prime Minister and Council of Ministers",
       "url": "https://opmcm.gov.np/",
       "description": "Nepal Government Office of the Prime Minister and Council of Ministers Official Site"
+    },
+    {
+      "name": "Nepal Police",
```
Comment (TheLearneer) [inline data/websites.json:54]: Please add this entry above in alphabetical order 🙇

Comment (TheLearneer) [reply data/websites.json:54]: @aCrazyPOTATO Please look into this issue as well 🙏

**Comment by aCrazyPOTATO** [comment]
Re-arranged in alphabetical order ✔️ @TheLearneer

**Review by TheLearneer — APPROVED** [review]
LGTM, thank you for your contribution @aCrazyPOTATO

**Pull request merged**